### ISSUE TITLE
Feature Request: Helm Context Menus

### ISSUE THREAD

**Issue by shreyu86** [Azure/vscode-kubernetes-tools#748]
It would be great if there are context menus on Helm Resources. 
Currently `Helm Releases` lists helm releases it would be really useful to have context menus like `Status`, `History`, `Rollback` `Uninstall` in the context menu of a helm release. 

Happy to submit a PR for this if this is something that would be considered helpful. 
If there is interest in adding the context menus, what would be a good place to start?

**Comment by itowlson** [comment]
I would love to see a PR for this - thanks!

In terms of where to start, not sure if you are asking about which command(s) to start with, or the technical aspects.  You've already done a lot of work on this extension already, so I'm guessing the former, because you probably know your way around the technical side!  (But happy to offer guidance if you do want it.)

I agree it would be best to PR just one or two initially, and then do the rest when we had worked through any design considerations.  My suggestions would be Rollback and Uninstall because those probably require less output than Status or History so you don't need to get tangled up in webviews or stuff...?

**Comment by shreyu86** [comment]
Thanks for the pointers @itowlson I will start with uninstall as the first context menu to add which gets me up to speed on the extension code base again once I gather feedback on the uninstall release function I can add rollback (I have a WIP for that struggling with `showQuickPick`)

**Comment by shreyu86** [comment]
opened https://github.com/Azure/vscode-kubernetes-tools/pull/749 as the first PR for the Helm context menus

**Comment by shreyu86** [comment]
For Helm release rollback I experimented with two implementation.
1.[Implementation with `showQuickPick` ](https://github.com/shreyu86/vscode-kubernetes-tools/tree/helm-rollback)
2. [Implementation with Helm history as a new node (child of Helm release node)](https://github.com/shreyu86/vscode-kubernetes-tools/tree/exp2)

Here is how the implementation1 look like: 

![impl1-lr](https://user-images.githubusercontent.com/311217/81524722-171c3480-9307-11ea-9636-06bd86ce0d13.gif)

and here is implementation 2 looks like:
![impl2-lr](https://user-images.githubusercontent.com/311217/81524742-2c915e80-9307-11ea-8783-f4b08b76797e.gif)

wanted to get early feedback before I open an MR so that I can polish the implementation of choice.

**Comment by lstocchi** [comment]
Difficult choice as they both make sense to me. 
I vote for the first (quickpick implementation) just for one reason. In the tree we currently show only items that have been actually created/deployed on the cluster. By showing all versions of a chart in the children nodes makes me think it can create confusion. A user could not know which is the current version used for that chart.

**Comment by shreyu86** [comment]
@lstocchi agreed I cannot decide which way to go either. Option #2 made more sense to me as it displayed all the releases in the context with the deployed, superseded labels. Is there anything we can do to improve both options and to get more feedback and pick the option to go with?

**Comment by lstocchi** [comment]
Let's wait some day and see if @itowlson answers, maybe he already have an idea on this. I guess he's very busy so hopefully beginning next week he will come back. I'll ping him.

**Comment by shreyu86** [comment]
@lstocchi Thanks for the feedback, I have opened https://github.com/Azure/vscode-kubernetes-tools/pull/753

**Comment by itowlson** [comment]
Aaugh, sorry everyone for not being around.  Thank you both for pushing ahead with it - this looks great!

**Comment by shreyu86** [comment]
@itowlson no worries. Keeping this open to add documentation, I should be able to open a PR sometime this week, if that is okay.

**Comment by webmutation** [comment]
Hi, will this add support for both helm v2 and v3 on the same panel? Currently I can only see releases for v2 in the helm panel.

**Comment by imdmahajankanika** [comment]
Hello, is it possible to extend the functionality of install option by allowing users to specify helm release name and set values or pass values.yaml during install?
